prediction_market_agent_tooling/markets/omen/omen.py

```
@@ -524,12 +511,12 @@ def get_positions(
         cls,
         user_id: str,
         liquid_only: bool = False,
-        larger_than: float = 0,
-    ) -> list[Position]:
+        larger_than: OutcomeToken = OutcomeToken(0),
```

Ruff (0.8.2)

Do not perform function call `OutcomeToken` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)